src/mlpack/tests/metric_test.cpp

```diff
@@ -133,4 +134,171 @@ BOOST_AUTO_TEST_CASE(IoUMetricTest)
   BOOST_REQUIRE_CLOSE(IoU<>::Evaluate(bbox1, bbox2), 0.7309670, 1e-4);
 }
 
+BOOST_AUTO_TEST_CASE(NMSMetricTest)
+{
+  arma::mat bbox, selectedBoundingBox, desiredBoundingBox;
+  arma::vec bbox1(4), bbox2(4), bbox3(4);
+  arma::uvec selectedIndices, desiredIndices;
+
+  // Set values of each bounding box.
+  // Use coordinate system to represent bounding boxes.
+  // Bounding boxes represent {x0, y0, x1, y1}.
+  bbox1 << 0.5 << 0.5 << 41.0 << 31.0;
+  bbox2 << 1.0 << 1.0 << 42.0 << 22.0;
+  bbox3 << 10.0 << 13.0 << 90.0 << 100.0;
+
+  // Fill bounding box.
+  bbox.insert_cols(0, bbox3);
+  bbox.insert_cols(0, bbox2);
+  bbox.insert_cols(0, bbox1);
+
+  // Fill confidence scores for each bounding box.
+  arma::vec confidenceScore(3);
```

Comment:
Picky comment, it needs to be 
```suggestion
  arma::vec confidenceScores(3);
```

Reply:
Done :)